### PR TITLE
DOCUMENTATIONS: Upgrade Example Code in Readme

### LINKED ISSUE #263: DOCUMENTATIONS: Upgrade Example Code in Readme
Model says ApiConfigurations, but the new one is OpenAIApiConfigurations

### PR DESCRIPTION
**Pull request by diogod3**
Closes #263

### CODE REVIEW

**Comment by Gummimundur** [comment]
@diogod3 There is another example in `README.md` that needs to be updated

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ namespace ExampleOpenAIDotNet
     {
         static async Task Main(string[] args)
         {
-            var openAIApiConfigurations = new ApiConfigurations
+            var openAIApiConfigurations = new OpenAIConfigurations
```
Comment (Gummimundur) [inline README.md:57]:
```suggestion
            var openAIConfigurations = new OpenAIConfigurations
```

Comment (Gummimundur) [reply README.md:57]: and update the parameter to OpenAIClient as well

**Comment by diogod3** [comment]
Good catch @Gummimundur

**Pull request merged**